app/models/user.rb

```diff
@@ -16,4 +16,8 @@ def role_names_for(application: )
   def roles
     memberships.map(&:roles).flatten
   end
+
+  def is_webops?
```

Comment:
I'm a bit concerned about scalability of this as we have more and more organisation / user types. But I guess we'll address it later when needed.